src/ol_infrastructure/infrastructure/aws/eks/Pulumi.infrastructure.aws.eks.applications.CI.yaml

```diff
@@ -29,12 +29,16 @@ config:
   eks:efs_csi_provisioner: "true"
   eks:gateway_release_channel: "experimental"
   eks:namespaces:
+  - "botkube"
   - "ecommerce"
   - "learn-ai"
   - "mitlearn"
+  - "mitx-edxapp"
+  - "mitx-staging-edxapp"
```

Comment:
These will want to live in a separate cluster since they don't share a scope with our other applications in the way that MITx Online and xPRO do.

Reply:
I don't follow.